[2023-10-25T10:48:28.648+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.get_csv manual__2023-10-25T10:48:27.603955+00:00 [queued]>
[2023-10-25T10:48:28.657+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.get_csv manual__2023-10-25T10:48:27.603955+00:00 [queued]>
[2023-10-25T10:48:28.658+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-10-25T10:48:28.672+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): get_csv> on 2023-10-25 10:48:27.603955+00:00
[2023-10-25T10:48:28.677+0000] {standard_task_runner.py:57} INFO - Started process 204 to run task
[2023-10-25T10:48:28.682+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'get_csv', 'manual__2023-10-25T10:48:27.603955+00:00', '--job-id', '100', '--raw', '--subdir', 'DAGS_FOLDER/first_dags.py', '--cfg-path', '/tmp/tmphzjufo6s']
[2023-10-25T10:48:28.683+0000] {standard_task_runner.py:85} INFO - Job 100: Subtask get_csv
[2023-10-25T10:48:28.722+0000] {task_command.py:416} INFO - Running <TaskInstance: ETL.get_csv manual__2023-10-25T10:48:27.603955+00:00 [running]> on host 80a803df81aa
[2023-10-25T10:48:28.806+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='me' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='get_csv' AIRFLOW_CTX_EXECUTION_DATE='2023-10-25T10:48:27.603955+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-25T10:48:27.603955+00:00'
[2023-10-25T10:48:28.820+0000] {base.py:73} INFO - Using connection ID 'postgres_docker' for task execution.
[2023-10-25T10:48:28.840+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/providers/common/sql/hooks/sql.py:219 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2023-10-25T10:48:28.889+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-25T10:48:28.902+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=get_csv, execution_date=20231025T104827, start_date=20231025T104828, end_date=20231025T104828
[2023-10-25T10:48:28.934+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-25T10:48:28.953+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
